lib/Config/Activator.php

```diff
@@ -2,22 +2,41 @@
 
 namespace MailPoet\Config;
 
+use MailPoet\InvalidStateException;
 use MailPoet\Settings\SettingsController;
+use MailPoet\WP\Functions as WPFunctions;
 
 class Activator {
+  public const TRANSIENT_ACTIVATE_KEY = 'mailpoet_activator_activate';
+  private const TRANSIENT_EXPIRATION = 120;
```

Comment:
Hey, could you leave a comment here on what units are we using? Is it 120 seconds or minutes?